python/pyspark/sql/functions.py

```diff
@@ -2713,6 +2713,25 @@ def from_csv(col, schema, options={}):
     return Column(jc)
 
 
+@since(3.0)
+def _getActiveSession():
```

Comment:
eh.. why is it in functions.py?

Reply:
Do you mean the _ prefix or the function itself?

Reply:
I mean the function itself ..